rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -478,6 +478,9 @@ public void registerBtcTransaction(Transaction rskTx, byte[] btcTxSerialized, in
                         sender,
                         co.rsk.core.Coin.fromBitcoin(totalAmount)
                 );
+
+                logger.info("Tranfering from BTC Address {}. RSK Address: {}.", senderBtcAddress, sender);
```

Comment:
there is a typo here: 
```suggestion
                logger.info("Transfering from BTC Address {}. RSK Address: {}.", senderBtcAddress, sender);
```

Reply:
I think it's "Transferring"